tools/proteomiqon_mzmltomzlite/proteomiqon_mzmltomzlite.xml

```diff
@@ -0,0 +1,292 @@
+<tool id="proteomiqon_mzmltomzlite" name="ProteomIQon MzMLToMzLite" version="0.0.5" python_template_version="3.5">
+    <requirements>
+        <requirement type="package" version="0.0.5">proteomiqon-mzmltomzlite</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        #import re
+        #set basename = $re.sub(r'[^\w ,.\-+]','_',$instrumentOutput.element_identifier)
+        ln -s $instrumentOutput '${basename}.mzML';
+        ln -s $out_mzlite '${basename}.mzlite';
+        proteomiqon-mzmltomzlite -i "./${basename}.mzML" -p $paramfile -o ./;
```

Comment:
please use for all paths and all text parameters single-quotes - this is for security reasons.